standalone/test/unit_tests/data_conversion_layer/unittest_monitoring_frame_serialization_deserialization.cpp

```diff
@@ -69,37 +126,40 @@ TEST(MonitoringFrameSerializationTest, shouldSerializeHexdumpFrameWithoutMeasure
 
 TEST(MonitoringFrameSerializationTest, shouldSerializeAndDeserializeFrameConsistently)
 {
-  std::array<data_conversion_layer::monitoring_frame::diagnostic::ErrorLocation, 3> error_locations = {
-    data_conversion_layer::monitoring_frame::diagnostic::ErrorLocation(0, 0),
-    data_conversion_layer::monitoring_frame::diagnostic::ErrorLocation(5, 0),
-    data_conversion_layer::monitoring_frame::diagnostic::ErrorLocation(4, 7)
+  std::array<monitoring_frame::diagnostic::ErrorLocation, 3> error_locations = {
```

Comment:
I think it would be reasonable to even use
```suggestion
  std::array<diagnostic::ErrorLocation, 3> error_locations = {
```
in this test? (with the respective `using` declaration)

Reply:
Nope. Other things from this namespace are also refered with `monitoring_frame` and I suggest to keeping it consistent. `monitoring_frame::Message` can also not be `Message` since this collides with `testing::Message`